spaceship/lib/spaceship.rb

```diff
@@ -1,3 +1,4 @@
+require 'fastlane_core'
```

Comment:
@KrauseFx  just to be sure, just revert all spaceship changes,remove require, and keep the old `ENV[]` stuff.?